test/testdata/lsp/completion/if_empty.rb

```diff
@@ -0,0 +1,12 @@
+# typed: true
+
+# TODO(jez) Fix this test
```

Comment:
This is going to be fixed in a subsequent change. I'll make a ticket for this eventually.